zds/gallery/views.py

```diff
@@ -387,7 +388,12 @@ def form_valid(self, form):
         data["title"] = form.cleaned_data.get("title")
         data["legend"] = form.cleaned_data.get("legend")
 
-        self.perform_update(data)
+        try:
+            self.perform_update(data)
+        except NotAnImage as e:
+            messages.error(self.request, _(f"Le fichier « {e} » n'est une image valide."))
```

Comment:
Lors de la modification d'une image, il affiche *Le fichier "titre de l'image" n'est pas une image valide." au lieu d'utiliser le titre du fichier.
```suggestion
            messages.error(self.request, _(f"Le fichier « {data['physical']} » n'est une image valide."))
```

Reply:
Oui c'est le cas actuellement, mais c'est corrigé dans mon autre PR à cet endroit : https://github.com/zestedesavoir/zds-site/pull/6207/files#diff-88bad9c03d50b38ffd5114cf1ecd89d4af10d0e73db43f21ecec552f1b309a29R332

J'ai hésité à mettre ce petit changement dans cette PR mais je me suis dit que ce n'était pas gênant et ça m'évite de devoir rebaser une PR après que l'autre soit fusionnée.

Reply:
```suggestion
            messages.error(self.request, _(f"Le fichier « {e} » n'est pas une image valide."))
```